packages/atomic/src/components/commerce/product-template/atomic-product-template.tsx

```diff
@@ -11,6 +11,14 @@ import {
 
 /**
  * @alpha
+ * * A [product template](todo) determines the format of the query results, depending on the conditions that are defined for each template.
+ *
+ * A `template` element must be the child of an `atomic-product-template`, and either an `atomic-commerce-product-list`, `atomic-commerce-recommendation-list` or `atomic-commerce-search-box-instant-products` must be the parent of each `atomic-product-template`.
+ *
+ * **Note:** Any `<script>` tags that are defined inside a `<template>` element will not be executed when the products are being rendered.
+ * @MapProp name: mustMatch;attr: must-match;docs: The field and values that define which product items the condition must be applied to. For example, a template with the following attribute only applies to product items whose `filetype` is `lithiummessage` or `YouTubePlaylist`: `must-match-filetype="lithiummessage,YouTubePlaylist"`;type: Record<string, string[]> ;default: {}
+ * @MapProp name: mustNotMatch;attr: must-not-match;docs: The field and values that define which product items the condition must not be applied to. For example, a template with the following attribute only applies to product items whose `filetype` is not `lithiummessage`: `must-not-match-filetype="lithiummessage";type: Record<string, string[]> ;default: {}
+ * @slot default - The default slot where to insert the template element.
```

Comment:
Can you explain what does `;attr:` and `;docs:` do precisely ? This looks new to me. Is this a different way to write JSdocs or is this something only for storybook ?

Reply:
Mh, I honestly don't fully know 🙃, I copy pasted from atomic-result-template

Reply:
I was curious so I did a bit of research. Turns out it's actually custom stuff we use for publishing the Atomic reference on the documentation site!

https://github.com/coveo/doc_jekyll-public-site/blob/master/scripts/GenerateAtomicDocs/generate_atomic_docs.rb#L158